### ISSUE TITLE
Monkey patching Array js example accuracy

### ISSUE THREAD

**Issue by HoffsMH** [turingschool/lesson_plans#111]
change this example to be more accurate

```javascript
 Array.prototype.strangePet = 'snake';

var myPets = ['dog', 'dog', 'rock'];
for(var i in myPets){
  console.log(i);
  // You will see 'snake' show up in your list of pets
}
```

**Comment by HoffsMH** [comment]
I will fix this!

**Comment by rrgayhart** [comment]
:+1:

**Comment by rrgayhart** [comment]
Fixed in #112
